src/api/routes.py

```diff
@@ -15,9 +15,15 @@ def get_masq(
     ua: Union[bool, None] = True,
     rf: Union[bool, None] = False,
     hd: Union[bool, None] = False,
+    count: Union[int, None] = 1,
```

Comment:
Let's add constraints here to set the count >0 and <= some limit to avoid extreme numbers.

I think there's a way to do it along these lines, but it would need to be tested:

```
from fastapi import APIRouter, Query
# ...
count: Union[int, None] = Query(default=1, gt=0, le=50)
```

I'm not sure what a suitable upper limit per API call is... 50 seems OK?

Reply:
Let's find a better way to make the limit dynamic / user specified, since I'm personally using it with 200 count and any fixed count limit will need to be dealt with.

I'm thinking something like an env variable - or something else that also works with docker.

Reply:
Out of curiosity how fast is the response with a 200 count compared with a single count, is there a noticeable delay in response?

Maybe something like a soft cap would work in that case: `if count > 250: count = 250` (or 500, 1000, etc).

It's easy enough to change the figure for anyone who wants to deploy the app themselves, so is just a consideriation for production. The production version times out when inactive, but I'll update that soon to keep it live.

Reply:
![Screenshot 2024-09-28 at 8 36 57 AM](https://github.com/user-attachments/assets/1a027e76-205c-47a9-8c80-a39032e645e0)

A little bit slower than eg: nodejs - golang, but it works.